pvr.hts/resources/settings.xml

```diff
@@ -15,6 +15,9 @@
     <setting id="autorec_approxtime" type="enum" label="30051" lvalues="30052|30053" default="0"/>
     <setting id="autorec_maxdiff" enable="eq(-1,1)"  type="slider" option="int" range="0,5,120" label="30054" default="15" />
     
+    <setting label="30500" type="lsep"/>
+    <setting id="streaming_profile" type="text" label="30501" default="" />
+
```

Comment:
I have the feeling that our "Advanced settings" page becomes to big. Is it just me or should we do a settings pages redesign soon?